packages/transit-vehicle-overlay/src/components/popups/vehicle-tracker.js

```diff
@@ -17,9 +17,9 @@ export default function VehicleTracker(props) {
   };
 
   return (
-    <PopupStyle.Button onClick={handleClick} className={cls}>
+    <S.Button onClick={handleClick} className={cls}>
```

Comment:
This reference isn't valid anymore because isn't directly exported from the style file, it needs to be `S.PopupStyle.Button` to work